images/consul/main.tf

```diff
@@ -0,0 +1,66 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+    oci  = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+variable "tag_suffix" {
+  description = "The suffix to the tag for this image."
+}
+
+variable "extra_repositories" {
+  type        = list(string)
+  default     = ["https://packages.wolfi.dev/os"]
+  description = "The list of additional repositories to append to the apko configuration."
+}
+
+variable "extra_keyring" {
+  type        = list(string)
+  default     = ["https://packages.wolfi.dev/os/wolfi-signing.rsa.pub"]
+  description = "The list of additional keyring entries to append to the apko configuration."
+}
+
+variable "extra_packages" {
+  type        = list(string)
+  default     = ["wolfi-baselayout"]
+  description = "The list of additional packages to append to the apko configuration."
+}
+
+provider "apko" {
+  extra_repositories = var.extra_repositories
+  extra_keyring      = var.extra_keyring
+  default_archs      = ["x86_64", "aarch64"]
+}
```

Comment:
Ultimately this is a key piece we will be able to remove with a top-level `main.tf` that configures these ~globally, but I'm opting for the "evil" of duplicating this boilerplate to let us have per-image control sooner than that.